Blog post of subcommunity resources (#427)
carpentries/carpentries.org: content/posts/2025/03/join-our-subcommunity-registry.md

---
layout: page

> authors: ["Angelique Trusler]

Author: ```suggestion
authors: ["Angelique Trusler"]

```

title: "Expanding Our Community: Subcommunity Invitations and Progress Update"
date: 2025-03-03
time: "09:00:00"
tags: ["Subcommunities", "Community"]
---

The **Community Development Program**, launched in 2022, continues to support The Carpentries’ growing network of **subcommunities**—groups within our larger community that are local (e.g., University Carpentries Clubs), regional (e.g., Carpentries in Africa, Carpentries in the UK), domain-specific (e.g., HPC Carpentry), language-based (e.g., Carpentries-es), or interest-driven (e.g., Carpentries Offline). Subcommunities built around formal Carpentries roles (e.g., Instructors, Maintainers) fall outside this definition. 

## Join Our Subcommunity Registry!

Our community requested a **[subcommunity registry](https://carpentries.org/community/get-connected/)** that captures ongoing activities, websites, and communication channels. We are excited to announce that our **[subcommunity registration form](https://docs.google.com/forms/d/e/1FAIpQLSd3BKF2KGPc_vv3vyuMxs8bqsSHwZvM3C1v6wXaKwVUV8rT4g/viewform?usp=send_form)** is open all year round! If you haven’t registered your subcommunity yet, now is the time.

> ### Why Register?

Author: ```suggestion
### _Why Register?_

```

The registry serves several important purposes:

* **Increase Visibility:** Highlight your subcommunity’s mission, events, and opportunities.
* **Connect with Members:** Help others find and engage with your subcommunity.
* **Recruit New Participants:** Welcome new members who share your interests.
* **Receive Targeted Support:** Enable the Community Engagement Team to understand your needs better and offer tailored assistance.

If you have any questions about registering, please [contact the **Community Engagement Team**](mailto:community@carpentries.org).

## Progress on Subcommunity Support

We’ve made significant strides in implementing additional community-driven recommendations. Here are some key updates:

> ### Quarterly Community Calls for Subcommunity Leaders and Member Organisations

Author: ```suggestion
### _Quarterly Community Calls for Subcommunity Leaders and Member Organisations_

```

We launched quarterly calls in 2022 to provide a space for networking and knowledge-sharing. Our next call is scheduled for **[11 March 2025 at 14:00 UTC and 23:00 UTC](https://pad.carpentries.org/community-development-program)**. If you’re involved in community building, we encourage you to attend!

> ### The Community Glossary and Carpentries Handbooks

Author: ```suggestion
### _The Community Glossary and Carpentries Handbooks_

```

The **[Community Glossary](https://github.com/carpentries/community-engagement/blob/main/glossary.md)** is available to clarify key terms. Additionally, we redeveloped our **[Carpentries Handbooks](https://docs.carpentries.org/index.html)** as part of our website redesign. 

> ### Communication Channels

Author: ```suggestion
### _Communication Channels_

```

We audited our **[Slack](https://slack-invite.carpentries.org/)** and **[TopicBox](https://carpentries.topicbox.com/latest)** channels in 2022, and this effort is ongoing. We expanded our social media presence by adding **[Mastodon](https://mastodon.social/@thecarpentries@hachyderm.io)** and **[Bluesky](https://bsky.app/profile/carpentries.carpentries.org)** accounts. 

> ### Event Registration for the Carpentries Community

Author: ```suggestion
### _Event Registration for the Carpentries Community_

```

---
layout: page
authors: ["Angelique Trusler"]

teaser: "Join Our Subcommunity Registry!"
title: "Expanding Our Community: Subcommunity Invitations and Progress Update"
date: 2025-03-03
time: "09:00:00"
tags: ["Subcommunities", "Community"]
---

> The **Community Development Program**, launched in 2022, continues to support The Carpentries’ growing network of **subcommunities**—groups within our larger community that are local (e.g., University Carpentries Clubs), regional (e.g., Carpentries in Africa, Carpentries in the UK), domain-specific (e.g., HPC Carpentry), language-based (e.g., Carpentries-es), or interest-driven (e.g., Carpentries Offline). Subcommunities built around formal Carpentries roles (e.g., Instructors, Maintainers) fall outside this definition. 

Reviewer: ```suggestion
The **Community Development Program**, launched in 2022, continues to support The Carpentries’ growing network of **subcommunities** within our larger community that are local (e.g., University Carpentries Clubs), regional (e.g., Carpentries in Africa, Carpentries in the UK), domain-specific (e.g., HPC Carpentry), language-based (e.g., Carpentries-es), or interest-driven (e.g., Carpentries Offline). Subcommunities built around formal Carpentries roles (e.g., Instructors, Maintainers) fall outside this definition. 
```

## Join Our Subcommunity Registry!

> Our community requested a **[subcommunity registry](https://carpentries.org/community/get-connected/)** that captures ongoing activities, websites, and communication channels. We are excited to announce that our **[subcommunity registration form](https://docs.google.com/forms/d/e/1FAIpQLSd3BKF2KGPc_vv3vyuMxs8bqsSHwZvM3C1v6wXaKwVUV8rT4g/viewform?usp=send_form)** is open all year round! If you haven’t registered your subcommunity yet, now is the time.

Reviewer: ```suggestion
Our community requested a **[subcommunity registry](https://carpentries.org/community/get-connected/)** that captures ongoing activities, websites, and communication channels. We are excited to announce that our **[subcommunity registration form](https://docs.google.com/forms/d/e/1FAIpQLSd3BKF2KGPc_vv3vyuMxs8bqsSHwZvM3C1v6wXaKwVUV8rT4g/viewform?usp=send_form)** is open all year round! If you have not registered your subcommunity yet, now is the time.
```

### _Why Register?_

The registry serves several important purposes:

* **Increase Visibility:** Highlight your subcommunity’s mission, events, and opportunities.
* **Connect with Members:** Help others find and engage with your subcommunity.
* **Recruit New Participants:** Welcome new members who share your interests.
* **Receive Targeted Support:** Enable the Community Engagement Team to understand your needs better and offer tailored assistance.

If you have any questions about registering, please [contact the **Community Engagement Team**](mailto:community@carpentries.org).

## Progress on Subcommunity Support

> We’ve made significant strides in implementing additional community-driven recommendations. Here are some key updates:

Reviewer: ```suggestion
We have made significant strides in implementing additional community-driven recommendations. Here are some key updates:
```
